python/sglang/srt/layers/moe/ep_moe/token_dispatcher.py

```diff
@@ -187,40 +193,83 @@ def dispatch_a(
         return hidden_states, topk_idx, topk_weights, previous_event
 
     def dispatch_b(self, hidden_states, topk_idx, topk_weights, previous_event):
-        (
-            hidden_states,
-            topk_idx,
-            topk_weights,
-            event,
-        ) = self._dispatch_core(hidden_states, topk_idx, topk_weights, previous_event)
-        event.current_stream_wait() if self.async_finish else ()
-        if hidden_states.shape[0] > 0:
-            reorder_topk_ids, seg_indptr, hidden_states = self._deepep_permute(
-                hidden_states, topk_idx, fp8_dtype=hidden_states.dtype
+        if _enable_jit_deepgemm:
+            # TODO hard code 128 block quant,use fp8 communication
+            if hidden_states.shape[0] > 0:
```

Comment:
This is a bypass related to `sglang_per_token_group_quant_fp8`. We can move this conditional statement to that function.